File: FairFaxTest/app/src/main/java/com/moji/fairfaxtest/domain/entities/NewsResponseView.java
package com.moji.fairfaxtest.domain.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

/**
 * Created by moji on 19/2/18.
 */

public class NewsResponseView implements Serializable {

    @SerializedName("assets")
    private List<NewsAssetView> assets;

    public List<NewsAssetView> getNewsViewList() {
        return assets;
    }
}

File: FairFaxTest/app/src/main/java/com/moji/fairfaxtest/domain/entities/NewsResponseView.kt
package com.moji.fairfaxtest.domain.entities

import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by moji on 19/2/18.
 * -------------------------------
 * NEWS API RESPONSE entity used for converting jSon string to objects
 */

class NewsResponseView : Serializable {

    @SerializedName("assets")
    val newsViewList: List<NewsAssetView>? = null
}
